airflow/contrib/operators/dataproc_operator.py

```diff
@@ -1108,17 +1114,17 @@ def __init__(self, template_id, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.template_id = template_id
 
-    def start(self):
+    def start(self, parameters):
```

Comment:
If would be better to add `self.parameters = parameters` in `__init__` because otherwise it will not work because `start` method is called by `execute` in `DataprocOperationBaseOperator`:
```python
    def execute(self, context):
        self.hook.wait(self.start())
```

Reply:
Thanks, I applied suggested changes, and I'm working on the unit tests.